modules/appeals_api/config/schemas/v1/10182_headers.json

```diff
@@ -15,10 +15,6 @@
         "X-VA-SSN":             { "$ref": "#/definitions/X-VA-SSN" },
         "X-VA-File-Number":     { "$ref": "#/definitions/X-VA-File-Number" },
         "X-VA-Birth-Date":      { "$ref": "#/definitions/X-VA-Birth-Date" },
-        "X-VA-Claimant-First-Name":     { "$ref": "#/definitions/X-VA-Claimant-First-Name" },
```

Comment:
A point of confusion - claimant headers still remain in nod v1 header schema, but no claimant data in form schema.